src/models/ui.ts

```diff
@@ -1,10 +1,13 @@
 import { types } from "mobx-state-tree";
 
+export type Spaces = "none" | "populations";
```

Comment:
@ekosmin and I discussed using an enumerated type for this (as is done in CLUE).  Might make the intellisense work better?  @ekosmin, do you have thoughts on this?

Reply:
I personally like what you did.  It's short and simple.  Just wanted to check if there is any reason to have CLUE and Cbio do the same thing in cases where we're "basically" trying to create an enumerated type.